var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -387,6 +388,12 @@ def configure_args(self):
         if spec.satisfies('@3.0.0:', strict=True):
             config_args.append('--with-zlib={0}'.format(spec['zlib'].prefix))
 
+        # Some scientific packages ignore deprecated/removed symbols.
+        # Re-enable them for now. See discussion at
+        # https://github.com/open-mpi/ompi/issues/6114#issuecomment-446279495
+        if spec.satisfies('@4.0:'):
```

Comment:
`spec.satisfies('@4.0')`

p..s they won't be there in 4.1

Reply:
scratch that, i think it does apply to `4.0.0`, right? In this case, it should be 

```
if spec.satisfies('@4.0.1:4.0.99'):
```

Reply:
Better `if spec.satisfies('@4.0.1:')` for the time being.
Yes, that flag was somehow broken in 4.0.0 and missed some symbols https://github.com/open-mpi/ompi/issues/6114